python/lsst/meas/astrom/matchOptimisticB.py

```diff
@@ -3,25 +3,20 @@
 from builtins import object
 import math
 
-import numpy as np
-
 import lsst.afw.table as afwTable
```

Comment:
I guess it still helps to run a linter rather then just relying on the highlighting in sublimetxt. It helps to pay attention huh?